### PR TITLE
Modified install.md file for Visual Studio Tools for Tizen

### PR DESCRIPTION
**Pull request by Reni-90**
### Change Description ###

Following are the changes:
-  Restructured the content for install.md file.
-  Added gifs for each process.
-  Added how to uninstall the tools section to complete the document.

If you want to see the previous content, see https://developer.tizen.org/development/visual-studio-tools-tizen/installing-visual-studio-tools-tizen.

Please note: If there are any issues with the image size or resolution, it will be fixed after all the topics are completed.

### CODE REVIEW

File: docs/application/vstools/install.md [hunk 1]
```diff
@@ -1,170 +1,102 @@
...
-- VSIX
-
-  Visual Studio extension for Tizen packaging
-
-- Tizen Baseline SDK
-
-  The SDK is shared with the Tizen Studio, and it supports tools (such as Certificate Manager, Device Manager, Emulator, SDB, and on-demand rpm) for developing Tizen .NET applications.
+# Introduction
 
+Visual Studio Tools for Tizen provides Tizen-specific tools to improve your productivity.
```
Comment (Reni-90) [inline docs/application/vstools/install.md]: Information Required: Could you provide some more details to give an overview.

File: docs/application/vstools/install.md [hunk 2]
```diff
@@ -1,170 +1,102 @@
...
-2. In the Visual Studio Marketplace, search for **Tizen**.
-
-   ![Marketplace](media/howtoinstall-extensionwindows.png)
-
-3. Click **Download** and close the Visual Studio IDE.
-
-   The installation starts.
-
-Alternatively, you can download the Visual Studio Tools for Tizen extension from the [Visual Studio Marketplace Web site](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen).
+## Install Visual Studio Tools for Tizen Extension
```
Comment (Reni-90) [inline docs/application/vstools/install.md]: Information Required: Could you provide some more details to give an overview. Why do we need to install, what is the purpose.

File: docs/application/vstools/install.md [hunk 3]
```diff
@@ -1,170 +1,102 @@
...
 
-1. In the Visual Studio IDE menu, go to **Tools &gt; Tizen &gt; Tizen Package Manager**.
-2. Select **Install new Tizen SDK**.
+![Visual Studio Tools for Tizen](media\v1-install-tools.gif)
 
-   ![Select new installation](media/howtoinstall-installwizard1.png)
+Alternatively, you can download the Visual Studio Tools for Tizen extension from [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen).
 
-3. Read the license document and click **I Agree**.
+## Install Tizen Baseline SDK
```
Comment (Reni-90) [inline docs/application/vstools/install.md]: Information Required: Could you provide some more details to give an overview. Why do we need to install?

File: docs/application/vstools/install.md [hunk 4]
```diff
@@ -1,170 +1,102 @@
...
+4.	Enter the root directory path where you want to install the Tizen baseline SDK and click **Next**. \
+    The baseline SDK is installed automatically and the Tizen Package Manager is launched.
+5.  In the Tizen Package Manager, go to **Main SDK**.
+6.  Install the **Tizen SDK tools**.
 
-   ![Installer download](media/howtoinstall-installwizard4.png)
+![Visual Studio Tools for Tizen](media\v2-install-baseline-sdk.gif)
 
-   ![Baseline SDK installation](media/howtoinstall-installwizard5.png)
+### Configure Existing Tizen Baseline SDK
```
Comment (Reni-90) [inline docs/application/vstools/install.md]: Information Required: Could you provide some more details to give an overview. Why do we need to install?

File: docs/application/vstools/install.md [hunk 5]
```diff
@@ -1,170 +1,102 @@
...
 
   ![Check the SDK tool path](media/howtoinstall-checktoolpath.png)
 
-- To check whether Tizen Project properties have been copied correctly, make sure that the `Tizen.NET.ProjectType.props` and `Tizen.NET.ProjectType.targets` files are located in the `C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\Tizen\VisualStudio` folder.
+- Tizen Project properties have been copied correctly. Ensure that the `Tizen.NET.ProjectType.props` and `Tizen.NET.ProjectType.targets` files are located in the `C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\Tizen\VisualStudio` folder.
 
   ![Project Property](media/cps-project-property.png)
 
+## Uninstall Visual Studio Tools for Tizen
+
```
Comment (Reni-90) [inline docs/application/vstools/install.md]: Information Required: Could you provide some more details to give an overview.

File: docs/application/vstools/install.md (hunk 1 above)
Comment (ayuckhulk) [reply docs/application/vstools/install.md]: Visual Studio Tools for Tizen allow to create, build and debug Tizen applications using Visual Studio IDE. With Visual Studio Tools for Tizen you can:
* Create applications for TV, Wearable and Mobile Tizen variants
* Debug your application from within Visual Studio
* Profile performance and memory usage of your application

File: docs/application/vstools/install.md (hunk 3 above)
Comment (ayuckhulk) [reply docs/application/vstools/install.md]: Baseline SDK contains Tizen-specific libraries and tools such as profilers and debugger.

**Comment by Reni-90** [comment]
@ayuckhulk and @manishtr 
Thank you for the review.

As a new PR, https://github.com/Samsung/tizen-docs/pull/949  is raised with the same content and also with the VS 2019 updates, closing this PR.

**Pull request closed without merging**